src/main/java/com/company/ApplicationConfig.java

```diff
@@ -0,0 +1,16 @@
+// ApplicationConfig.java
+package com.company;
+
+import com.company.service.FileReaderService;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class ApplicationConfig {
+
+    @Bean
+    public FileReaderService fileReaderService() {
+        return new FileReaderService();
+    }
```

Comment:
The configuration class seems straightforward, creating a bean for FileReaderService. 🎉 No issues found. 👍🏻